unity-client/Assets/ABConverter/Core.cs

```diff
@@ -83,9 +83,14 @@ public Core(Environment env, Client.Settings settings = null)
             public void Convert(ContentServerUtils.MappingPair[] rawContents, Action<ErrorCodes> OnFinish = null)
             {
                 if (OnFinish == null)
+                {
                     OnFinish = CleanAndExit;
+                }
                 else
+                {
+                    OnFinish -= CleanAndExit;
                     OnFinish += CleanAndExit;
+                }
```

Comment:
This doesn't work if we just do 

` OnFinish += CleanAndExit;`?

Reply:
You are right! I just tested it just in case, but this works fine with just that one line.